templates/base/config/api/index.js

```diff
@@ -0,0 +1 @@
+export const graphQLApiUri = 'https://48p1r2roz4.sse.codesandbox.io';
```

Comment:
@bhardwajamit0890 this will have to change as we are one of other PR setting up a config/connections.js to hold all such information